validator/src/main/java/com/amazon/aoc/helpers/CWMetricHelper.java

```diff
@@ -20,57 +20,130 @@
 import com.amazon.aoc.models.Context;
 import com.amazonaws.services.cloudwatch.model.Dimension;
 import com.amazonaws.services.cloudwatch.model.Metric;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
```

Comment:
Any specific reason why we opt to use the wildcard import here? Small nit which I don't think matters in this context, but for example I believe this would cause a spotbugs warning when compiling. I think this applies to a few other flies' code imports as well